Common/src/main/java/com/fren_gor/ultimateAdvancementAPI/UltimateAdvancementAPI.java

```diff
@@ -259,9 +259,9 @@ public void displayCustomToast(@NotNull Player player, @NotNull ItemStack icon,
      *
      * @throws RuntimeException If the operation fails. It is a wrapper for the real exception.
      */
-    public void disableVanillaAdvancements() throws RuntimeException {
+    public void disableVanillaAdvancements(boolean disableVanillaAdvancementsRecipes) throws RuntimeException {
```

Comment:
This is a breaking change, please *also* keep the old method without the parameter (passing `true` as default value). Additionally, please add the description of the parameter in the javadoc.

(Please fix this also in the other classes)

Reply:
Done in 9703f3b628423c736e0a5060ccc43896d030dddc as you suggested in an other comment